x-pack/metricbeat/module/statsd/server/data_test.go

```diff
@@ -8,14 +8,616 @@ import (
 	"testing"
 	"time"
 
+	"github.com/elastic/beats/v7/metricbeat/mb"
+
+	"gopkg.in/yaml.v2"
+
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
 	"github.com/elastic/beats/v7/libbeat/common"
 	"github.com/elastic/beats/v7/metricbeat/helper/server"
 	mbtest "github.com/elastic/beats/v7/metricbeat/mb/testing"
+
+	_ "github.com/elastic/beats/v7/libbeat/processors/actions"
```

Comment:
Processors shouldn't be needed to test this module, import them only in the airflow tests.

Reply:
This is needed for generating `data.json`